q2_diversity/plugin_setup.py

```diff
@@ -33,6 +33,19 @@
             'phylogeny': Phylogeny[Rooted]},
     parameters={'metric': Str % Choices(beta.phylogenetic_metrics())},
     outputs=[('distance_matrix', DistanceMatrix % Properties('phylogenetic'))],
+    input_descriptions={
+        'table': ('The feature table containing the samples over which beta '
+                  'diversity should be computed.'),
+        'phylogeny': ('Phylogenetic tree containing tip identifiers that '
+                      'correspond to the feature identifiers in table. This '
+                      'tree can contain tip ids that are not present in '
+                      'table, but all feature ids in table must be present '
```

Comment:
Using "the table" or backticking `table` would make this description easier to read. Right now "table" is being used in a few places and it looks like grammatical error.